test/issuehunter.js

```diff
@@ -491,6 +491,39 @@ contract('Issuehunter', function (accounts) {
       })
     })
 
+    context('fund overflow', function () {
+      const issueId = nextCampaignId()
+      const funder = sampleAccount()
+      const txValue1 = (2 ** 32) - 1
+      const txValue2 = 1
+
+      const initialTotal = issuehunter.then(function (instance) {
+        return instance.campaigns.call(issueId)
+      }).then(function (campaign) {
+        return campaign[1].toNumber()
+      })
+
+      return newCampaign(issueId, sampleAccount()).then(function () {
+        return initialTotal
```

Comment:
You don't need this callback, do you?

Reply:
Not following, what do you mean exactly?

Reply:
I mean that you're not using the `initialTotal` return value, so the first `then` seems useless to me.